### PR TITLE
Add 'local' to list of logging drivers that support `docker-compose logs`

### LINKED ISSUE #7420: `docker-compose logs` doesn't work for local log driver
## Context information (for bug reports)

**Output of `docker-compose version`**
```
docker-compose version 1.25.5, build 8a1c60f6
docker-py version: 4.1.0
CPython version: 3.7.5
OpenSSL version: OpenSSL 1.1.0l  10 Sep 2019

```

**Output of `docker version`**
```
Client:
 Version:           18.09.9-ce
 API version:       1.39
 Go version:        go1.10.3
 Git commit:        039a7df
 Built:             Fri Nov  1 19:26:49 2019
 OS/Arch:           linux/amd64
 Experimental:      false

Server:
 Engine:
  Version:          18.09.9-ce
  API version:      1.39 (minimum version 1.12)
  Go version:       go1.10.3
  Git commit:       039a7df
  Built:            Fri Nov  1 19:28:24 2019
  OS/Arch:          linux/amd64
  Experimental:     false

```

**Output of `docker-compose config`**
(Make sure to add the relevant `-f` and other flags)
```
services:
  rabbit:
    image: rabbitmq
    logging:
      driver: local
version: '3.0'
```

## Steps to reproduce the issue

1. Create the above docker-compose.yaml file
2. Call `docker-compose up -d`
3. Call `docker-compose logs`

### Observed result
$ docker-compose logs 
Attaching to mike_rabbit_1
rabbit_1  | WARNING: no logs are available with the 'local' log driver

### Expected result
Get logs from rabbitmq

### Stacktrace / full error message

```
N/A
```

## Additional information
If I do `docker logs <container-id>` this works.

This seems like it comes from `compose/container.py`:
```
    @property
    def has_api_logs(self):
        log_type = self.log_driver
        return not log_type or log_type in ('json-file', 'journald')
```
That doesn't include 'local'.

I'll try to fix it and if it's as easy as adding 'local' to the list - will send a PR tomorrow.

### PR DESCRIPTION
**Pull request by svarogg**
Resolves #7420

### CODE REVIEW

File: compose/container.py [hunk 1]
```diff
@@ -193,7 +193,7 @@ def log_driver(self):
     @property
     def has_api_logs(self):
         log_type = self.log_driver
-        return not log_type or log_type in ('json-file', 'journald')
+        return not log_type or log_type in ('json-file', 'journald', 'local')
```
Comment (thaJeztah) [inline compose/container.py:196]: Slightly related; looks like the description here also needs updating; https://github.com/docker/compose/blob/4bc1cbc32accce59acddece7314eab6929eefe42/compose/container.py#L211

Comment (svarogg) [reply compose/container.py:196]: Thanks for the tip. Updated the comment.

**Comment by solomson** [comment]
Will this also fix #7408 ?

**Comment by avkonst** [comment]
Hey, can this PR be merged and new compose version released with the fix?

**Comment by avkonst** [comment]
Thanks for the merge! Is there any information when it is going to be released in a new version?

**Comment by thaJeztah** [comment]
Not sure when the next release will be (@rumpl may know if something is planned already).


Just a heads up; the next Docker (20.xx) release will include a feature called "dual logging" (docs: https://docs.docker.com/config/containers/logging/dual-logging/) that was previously only in the Enterprise Edition of the Docker Engine. That feature has been upstreamed (through https://github.com/moby/moby/pull/40543). With that feature, the "local" logging driver can be always enabled, which means that the `logs` endpoint will always be able to read back logs from a buffer, irregardless of the logging driver that is used (which could be a logging plugin that sends logs to a remote logging aggregator)

**Pull request merged**